docs/src/calibration_features.md

```diff
@@ -1,28 +1,59 @@
+- **Observations**: The reference data against which simulations are compared. To do this, you can select the source of data ("perfect_model" or "file"), as well as the data names, data type, and the characteristics of the validation set for the calibrations.
+- **Calibration Process**: The methods and approaches used for calibration. In this part, the user must select the calibration method, as well as the variables that are needed for the process (e.g., number of iterations, batch size, random seed, etc.).
+- **Model Parameters setup**: Details about the parameters that affect the model's behavior, such as the model name, the characteristics of the grid, the aerosol initial distribution, the precipitation choice, etc.
+- **Statistics**: The statistical methods that can be applied on the data and simulation results to compare them.
+
+## How to run a calibration
+To be able to run a calibration, you need to import the `/KinematicDriver/CalibrateCMP` folder. 
+## Example
+To better understand how to build the configuration file and run calibrations in KiD, you can find a 1-dimensional calibration example in the `test/experiments/calibrations` folder.  In this example, we calibrate against the PySDM products of rain water mixing ratio (``r_r``) and cloud water mixing ratio (``r_l``), and KiD uses the [1-moment microphysics bulk scheme](https://clima.github.io/CloudMicrophysics.jl/dev/Microphysics1M/) to compute the same products. The parameters that are calibrated are ``\chi_v``, and ``\chi_a``, two parameters that appear in the terminal velocity (``v_{term}(r)``) and rain cross-section (``a(r)``) formulas of the bulk scheme:
+```math
+a(r) = \chi_a \, a_0 \left(\frac{r}{r_0}\right)^{a_e + \Delta_a}
+```
+```math
+v_{term}(r) = \chi_v \, v_0 \left(\frac{r}{r_0}\right)^{v_e + \Delta_v}.
+```
+
+Inside the calibration example, there are examples of the `config.jl` file, the `run_calibration.jl` file, and `run_model.jl`.
+
+In `config.jl`, we define the function `get_config()`, which creates the configuration dictionary. The keys of this dictionary are associated with all the elements that are needed for the calibration process, and they are retrieved using the following functions:
+- `get_prior_config()`: Sets up the priors' configuration.
+- `get_observations_config()`: Specifies the reference data.  
+- `get_process_config()`: Defines the method for the calibration and the associated parameters.
+- `get_model_config()`: Defines the model setup.
+- `get_stats_config()`: Specifies the statistical methods used on the reference data and simulation results.
+
+After creating the configuration file, running the calibrations using the `run_calibration.jl` file becomes very straightforward. 
```

Comment:
After creating the configuration file, running the calibrations can be performed by using the `run_calibration.jl` file.